scripts/models/portfolio.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+(function(module) {
+
+  function Project (obj) {
+    this.title = obj.title;
+    this.dateCreated = obj.dateCreated;
+    this.image = obj.image;
+    this.description = obj.description;
+    this.fileSource = obj.fileSource;
+  }
+
+  Project.allProjects = [];
+
+  Project.prototype.toHtml = function() {
+    var htmlTemp = $('#template').html(); //#template refers to the ID of the Handlebars script tag right above the HTML template
+    var temp = Handlebars.compile(htmlTemp); //Handlebars.compile returns a function, so we store it in a variable, pass 'this' into it (next line), and return it.
+    return temp(this);
+  };
+
+  Project.loadAll = function(projects) {
+    projects.forEach(function(ele) {
+      Project.allProjects.push(new Project(ele));
+    });
+  };
+
+  Project.fetchAll = function() {
+    if (localStorage.projectInfo) {
+      var lsProject = JSON.parse(localStorage.getItem('projectInfo'));
+      Project.loadAll(lsProject);
+    } else {
+      $.getJSON('../../data/projectInfo.json').then(
+        function(data) {
+          localStorage.setItem('projectInfo', JSON.stringify(data));
+          Project.loadAll(data);
```

Comment:
You're correctly storing your project data in local storage and it's being loaded when it's already there, but it's not being displayed on the page the first time you visit your page. If you type localStorage.clear() in your browser console and then revisit your page, you'll see this. Perhaps you need to call your render function? You'll need to wrap the projectView file in an IIFE and attach your functions to a global object inside of that IIFE that gets added to the window.